scrapy/czlscrape/utils.py

```diff
@@ -46,3 +51,13 @@ def strip_diacritics(text: str) -> str:
 
 def romanian_month_number(text: str) -> int:
     return ROMANIAN_MONTHS.get(text.lower())
+
+
+def extract_documents(selector: SelectorList):
```

Comment:
Mi se pare un pic forțată interfața funcției ăsteia (ce primește ca argumente și ce returnează) dar n-am o sugestie mai bună.